.env.template

```diff
@@ -1,16 +0,0 @@
-SERVICE_NAME="tmsService"
```

Comment:
I think this file should stay here?

Reply:
there are 2 properties - FunctionName and ServiceName - but what's the different? So I changed the code that everything uses FunctionName instead of ServiceName, so this property is now obsolete.